lib/EasyPost/Billing.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace EasyPost;
+
+/**
+ * @package EasyPost
+ * @property string $id
+ * @property string $object
+ * @property Object $primary_payment_method
+ * @property Object $secondary_payment_method
+ */
+class Billing extends EasypostResource
+{
+    /**
+     * Retrieve all payment methods.
+     *
+     * @param mixed $params
+     * @param string $apiKey
+     * @return mixed
+     */
+    public static function retrieve_payment_methods($params = null, $apiKey = null)
+    {
+        $payment_methods = self::_all("paymentMethod", $params, $apiKey);
+
+        if ($payment_methods->id == null) {
+            throw new Error('Billing has not been setup for this user. Please add a payment method.');
+        }
+        return $payment_methods;
+    }
+
+    /**
+     * Fund your EasyPost wallet by charging your primary or secondary payment method.
+     *
+     * @param string $amount
+     * @param string $primary_or_secondary
+     * @param string $api_key
+     * @return mixed
+     */
+    public static function fund_wallet($amount, $primary_or_secondary = 'primary', $api_key = null)
+    {
+        $payment_method_info = Billing::get_payment_info($primary_or_secondary);
+        $payment_method_endpoint = $payment_method_info[0];
+        $payment_method_id = $payment_method_info[1];
+
+        $url = $payment_method_endpoint . "/$payment_method_id/charges";
+        $wrapped_params = ['amount' => $amount];
+        $requestor = new Requestor($api_key);
+        list($response, $api_key) = $requestor->request('post', $url, $wrapped_params);
+        return Util::convertToEasyPostObject($response, $api_key);
+    }
+
+    /**
+     * Delete a payment method.
+     *
+     * @param string $primary_or_secondary
+     * @param string $api_key
+     * @return mixed
+     */
+    public static function delete_payment_method($primary_or_secondary, $api_key = null)
+    {
+        $payment_method_info = Billing::get_payment_info($primary_or_secondary);
+        $payment_method_endpoint = $payment_method_info[0];
+        $payment_method_id = $payment_method_info[1];
```

Comment:
I believe we can do this here:

```suggestion
        [$payment_method_endpoint, $payment_method_id] = Billing::get_payment_info($primary_or_secondary);
```

Reply:
resolved